nautobot/ui/src/components/Retrieve/RenderTable.js

```diff
@@ -3,6 +3,14 @@ import { Table, Tbody, Td, Tr } from "@nautobot/nautobot-ui";
 import { ObjectTableItem } from "@components";
 import { buildUrl } from "@utils/url";
 
+function construct_model_url(model_title) {
+    // helper function to turn plural model names to url safe strings
+    // e.g. tenant groups -> tenant-groups
```

Comment:
Are we 100% sure this would always be the case for urls? 𖡄

Reply:
Why don't we just return the appropriate url string from the backend, instead of trying to construct it?